app/static/src/app/components/WodinSession.vue

```diff
@@ -1,13 +1,23 @@
 <template>
-  <router-view v-if="initialised"></router-view>
+  <session-initialise-modal :open="appInitialised && !sessionInitialised && !isSessionsRoute"
```

Comment:
would it not be simpler to just replace `:open` with `v-if` and assume true in modal component? it would probably also simplify testing a bit and simplify that component

Reply:
Yeah, I think we got into the habit of using this pattern because of an example where we didn't want to re-render the modal every time we showed it because of some reactivity issue (can't remember the details) but should be ok to simplify in this case I think